fog/overseer/server/src/config.rs

```diff
@@ -11,20 +11,15 @@ use structopt::StructOpt;
 #[derive(Clone, Serialize, StructOpt)]
 pub struct OverseerConfig {
     /// Host to listen on.
-    #[structopt(long, default_value = "127.0.0.1")]
+    #[structopt(long, env, default_value = "127.0.0.1")]
```

Comment:
If we intend to use this as an environment variable, does it make sense to give it a more specific name to make it less likely to conflict?

Reply:
sgtm, done.